src/sass/_pattern_navigation.scss

```
@@ -10,10 +10,14 @@
   }
 
   .logo-text {
-    color: #fff;
+    color: $colors--light-theme--background-default;
     left: 47px;
     position: absolute;
     top: $spv--x-small;
+
+    &.is-light {
+      color: $colors--dark-theme--background-default;
```

```suggestion
      color: $colors--light-theme--text-default;
```
Better use the font variables here, it's not a background.